api/api-doc/storage_service.json

```diff
@@ -2164,7 +2164,42 @@
                ]
             }
          ]
-      }
+      },
+      {
+         "path":"/storage_service/sstable_info",
+         "operations":[
+            {
+               "method":"GET",
+               "summary":"SSTable information",
+               "type":"array",
+               "items":{
+                  "type":"table_sstables"
+               },
+               "nickname":"sstable_info",
+               "produces":[
+                  "application/json"
+               ],
+               "parameters":[
+                  {
+                     "name":"keyspace",
+                     "description":"The keyspace",
+                     "required":false,
+                     "allowMultiple":false,
+                     "type":"string",
+                     "paramType":"query"
+                  },
+                  {
+                     "name":"table",
```

Comment:
while it makes sense using table (as we are following Cassandra's naming) everything else is CF and even your description says Column Family.

I would go with using `cf` as the name, but even with `table` the description should match

Reply:
Honestly, I would rather clean up the description. We should really migrate to "table" (and potentially allow using either name for the parameter in all places).

Reply:
Changed to cf